lmms_eval/models/chat/thyme.py

```
@@ -0,0 +1,443 @@
+    """
+    user_image = None
+
+    for message in message_list:
+        if message.get("role") == "user":
+            content = message.get("content", [])
+            for part in content:
+                if part.get("type") == "image":
+                    user_image = part.get("image", None)
+
+    if user_image is None:
+        raise ValueError("Not found valid image.")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)